.github/workflows/ci_workflows.yml

```
@@ -86,6 +83,12 @@ jobs:
           - enchant
           - hdf5
       envs: |
+
+        # Non-deterministic segmentation faults
```

```suggestion
        # Non-deterministic QThread exceptions
```
Those, and the macos-pyside63 one, are actually all failing on the
> QThread: Destroyed while thread is still running
test return, but that seems to be non-deterministic nonetheless (timing issue?)